src/cargo/core/shell.rs

```diff
@@ -130,7 +121,7 @@ impl Shell {
     /// Creates a shell from a plain writable object, with no color, and max verbosity.
     pub fn from_write(out: Box<dyn Write>) -> Shell {
         Shell {
-            output: ShellOut::Write(out),
+            output: ShellOut::Write(AutoStream::never(out)),
```

Comment:
I'm assuming the `AutoStream::never` here is to ensure we strip out formatting? Might be worth a comment.